src/Perlang.Tests/ConsoleApp/ProgramTest.cs

```diff
@@ -172,31 +181,46 @@ public class WithPrintParameter
 
                 /// <summary>
                 /// Gets the content printed to the standard output stream during test execution.
+                ///
+                /// This property returns the output as one, long string.
                 /// </summary>
                 private string StdoutContent => testConsole.Out.ToString() ?? String.Empty;
 
+                /// <summary>
+                /// Gets the content printed to the standard output stream during test execution.
+                ///
+                /// This property returns the output split by newline separators.
+                /// </summary>
+                private IEnumerable<string> StdoutLines => StdoutContent.Split(Environment.NewLine);
+
                 [Fact]
                 public void assignment_and_increment()
                 {
                     CallWithPrintParameter("i = i + 1");
 
-                    Assert.Equal("(i (+ i 1))\n", StdoutContent);
```

Comment:
This was the problem this MR fixed: there were hardwired assumptions on `Environment.NewLine` being `\n` (LF), which is the case on Linux and macOS but definitely not on Windows. By converting this to checking the `StdoutLines` (added in this MR), we both fixed this and made it more convenient to write these assertions at the same time (since we can now check an `IEnumerable` of `string`s instead of a long `string`).